exonum/src/node/state.rs

```diff
@@ -647,16 +641,16 @@ impl State {
 
     /// Increments node's round by one.
     pub fn new_round(&mut self) {
-        self.round += 1;
+        self.round.increment();
     }
 
     /// Increments the node height by one and resets previous height data.
     // FIXME use block_hash
     pub fn new_height(&mut self, block_hash: &Hash, height_start_time: SystemTime) {
-        self.height += 1;
+        self.height.increment();
         self.height_start_time = height_start_time;
-        self.round = 1;
-        self.locked_round = 0;
+        self.round = Round(1);
```

Comment:
Just for one call? :smile: Additionally, in sandbox tests different rounds are used.